src/codeflare_sdk/cluster/config.py

```diff
@@ -46,7 +46,7 @@ class ClusterConfiguration:
     num_gpus: int = 0
     template: str = f"{dir}/templates/base-template.yaml"
     instascale: bool = False
-    mcad: bool = True
+    mcad: bool = False
```

Comment:
Do we need to worry about this being a breaking change?

Reply:
I don't think so. Unless user's of the SDK are not made aware of how Kueue will be mandatory if you are not using MCAD.

The unit tests and e2es are updated to reflect the change.